client/src/pages/Home/Home.js

```diff
@@ -8,7 +8,9 @@ class Home extends PureComponent {
 
   render() {
     return (
-      <div className="Home container">
+      //ensure all css classes are lowercase - best practices
+      //this is the container for all things within the article App
+      <div className="home container">
```

Comment:
With CSS modules we cannot use lowercase naming convention.
Since these names are used inside Javascript where "`-` "is not allowed.
naming has to be in the form `flatButton` or `FlatButton`.
But we cannot use `flat-button`. Using no spectator for words is **not** preferred eg `flatbutton`.

Reply:
We should be using css modules in Home.js.
The index.css will be the only global css file correct?

Reply:
Yes correct. And we are using separate css files for each component. But not using `css-modules` per se. It's just like writing normal css.

Reply:
oh I see, I was under the impression that cra was pre configured with css modules.
Then lowercase naming is fine by me.

Reply:
It is sort of pre configured with css-modules but uses only autoprefixer.

Sure, shall we merge this PR then? If all approves.

Reply:
I did not know about the naimng conventions within react per se. 
However, If it is only one single word, it can be lowercase. 
the issue you might be referring to @anantoghosh , might happen, only when using two or more words in a name. In which case, we can always have camel / snake cases as react expects it to be in case we use a module.

Reply:
So, to overcome the issue, I used to use `_`s instead of `-`s

Reply:
And by the way there are many different style guides out there to name css classes. It's not fixed to one being referenced here. So I think we can stick with what we are following.
`.ComponentName_heading` and `.ComponentName_container` and stuff life that.

There is a reason behind what we are following: We avoid overlaps in name. Suppose we want to have .container class in two component which would overlap. Prefixing with component's name would resolve that.

I know there are lots of other ways css-modules adds a hash to the classname to prevent this and other solutions but we are following this and I think we should stick with this.

Do let me know your thoughts: @retrocausal and @anantoghosh